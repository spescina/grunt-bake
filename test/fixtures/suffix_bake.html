<!DOCTYPE html>
<html>
	<head></head>
	<body>
		<!--(bake includes/book.html suffix{"_two"})-->
	</body>
</html>