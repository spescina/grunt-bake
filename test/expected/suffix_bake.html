<!DOCTYPE html>
<html>
	<head></head>
	<body>
		<h1>test string</h1>
		<div>Apple Pie</div><span>Mathias</span>
		<h1>test string modified</h1>
		<div>Another Apple Pie</div><span>Mathias Another</span>
		<h1>huhu</h1>
		<div>Another Apple</div><span>Another</span>
	</body>
</html>